server/app.js

```diff
@@ -3,6 +3,7 @@ const mongoose = require("mongoose");
 const routes = require("./routes/index");
 const helpers = require("./helpers");
 const path = require('path');
+const errorHandlers = require('./handlers/errorHandlers');
```

Comment:
Is this being used at all?

Reply:
updated, as it's not being used at the moment.